#! /bin/sh /usr/share/dpatch/dpatch-run
## 04.fix_manpages.dpatch by Rob Weir <rweir@ertius.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use \- instead of - when indicating minus signs in manpages.

@DPATCH@
diff -urNad lyx-1.3.6/lyx.man /tmp/dpep.oxXQ25/lyx-1.3.6/lyx.man
--- lyx-1.3.6/lyx.man	2005-09-13 20:09:50.000000000 +1000
+++ /tmp/dpep.oxXQ25/lyx-1.3.6/lyx.man	2005-09-13 20:58:41.703379976 +1000
@@ -58,15 +58,15 @@
 .TP
 .BI \-dbg " feature[,feature...]"
 where feature is a name or number.
-Use "\fBlyx -dbg\fR" to see the list of available debug features.
+Use "\fBlyx \-dbg\fR" to see the list of available debug features.
 .TP
-\fB -x [--execute]\fP \fIcommand
+\fB \-x [\-\-execute]\fP \fIcommand
 where command is a lyx command.
 .TP
-\fB -e [--export]\fP \fIfmt
+\fB \-e [\-\-export]\fP \fIfmt
 where fmt is the export format of choice.
 .TP
-\fB -i [--import]\fP \fIfmt file.xxx
+\fB \-i [\-\-import]\fP \fIfmt file.xxx
 where fmt is the import format of choice and file.xxx is the file to be imported.
 
 .SH XFORMS FRONTEND OPTIONS
@@ -131,7 +131,7 @@
 The system directory is determined by searching for the file
 "chkconfig.ltx". Directories are searched in this order:
 .br
-1) -sysdir command line parameter
+1) \-sysdir command line parameter
 .br
 2) LYX_DIR_13x environment variable
 .br
@@ -146,7 +146,7 @@
 .PP
 The user directory is, in order of precedence:
 .br
-1) -userdir command line parameter
+1) \-userdir command line parameter
 .br
 2) LYX_USERDIR_13x environment variable
 .br
